src/app/pages/vm/vm-cards/vm-cards.component.ts

```diff
@@ -522,12 +523,22 @@ export class VmCardsComponent implements OnInit,OnDestroy {
     }
      else {
       if(poweroff){
-        eventName = "VmPowerOff";
-      } else {
-        eventName = "VmStop";
+         this.off_text =  `Do you wish to Power Off ${vm.name} ?`;
```

Comment:
Please do not use "you", and no reason to capitalize "power off" specially.  Also, set off the name of the VM so the user can see it is a name:
`` this.off_text =  `Turn off power to virtual machine "${vm.name}"?`; ``